integration/spark/src/main/scala/org/apache/spark/sql/hive/CarbonMetastoreCatalog.scala

```diff
@@ -101,8 +101,8 @@ case class DictionaryMap(dictionaryMap: Map[String, Boolean]) {
   }
 }
 
-class CarbonMetastoreCatalog(hive: HiveContext, val storePath: String, client: ClientInterface)
-  extends HiveMetastoreCatalog(client, hive)
+class CarbonMetastoreCatalog(hiveContext: HiveContext, val storePath: String, client: ClientInterface)
+  extends HiveMetastoreCatalog(client, hiveContext)
```

Comment:
check style is failing here. Please do `mvn scalastyle:check` and `mvn checkstyle:check`